flycheck.el

```diff
@@ -7182,7 +7182,7 @@ See Info Node `(elisp)Byte Compilation'."
 
 Variables are taken from `flycheck-emacs-lisp-checkdoc-variables'."
   `(progn
-     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(symbol-value opt)))
+     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(custom-quote (symbol-value opt))))
```

Comment:
I see.

But it _will hurt_ for example:
```elisp
(if 'nil false t)    
```
I suspect this is the reason why `custom-quote` exists.